nemo/collections/nlp/nm/trainables/common/huggingface/huggingface_utils.py

```diff
@@ -0,0 +1,55 @@
+# =============================================================================
```

Comment:
why did you move this to nm.trainables, and not leave it in utils?

Reply:
had import loop issues and this file is needed only for huggingface trainables